Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Jamulus/Startpage](https://hosted.weblate.org/projects/jamulus/1-index/).


It also includes the following components:

* [Jamulus/Installation-for-Android](https://hosted.weblate.org/projects/jamulus/installation-for-android/)

* [Jamulus/Getting-Started](https://hosted.weblate.org/projects/jamulus/getting-started/)

* [Jamulus/Contribution](https://hosted.weblate.org/projects/jamulus/contribution/)

* [Jamulus/Directories](https://hosted.weblate.org/projects/jamulus/directories/)

* [Jamulus/Installation-for-Macintosh](https://hosted.weblate.org/projects/jamulus/installation-for-macintosh/)

* [Jamulus/Client-Troubleshooting](https://hosted.weblate.org/projects/jamulus/client-troubleshooting/)

* [Jamulus/Installation-for-Windows](https://hosted.weblate.org/projects/jamulus/installation-for-windows/)

* [Jamulus/Installation-for-Linux](https://hosted.weblate.org/projects/jamulus/installation-for-linux/)

* [Jamulus/Include-Server-Commands](https://hosted.weblate.org/projects/jamulus/include-server-commands/)

* [Jamulus/Command-Line-Options](https://hosted.weblate.org/projects/jamulus/command-line-options/)

* [Jamulus/General strings](https://hosted.weblate.org/projects/jamulus/general/)

* [Jamulus/QOS-Windows](https://hosted.weblate.org/projects/jamulus/qos-windows/)

* [Jamulus/Server-Bandwidth](https://hosted.weblate.org/projects/jamulus/server-bandwidth/)

* [Jamulus/Installation-for-iOS](https://hosted.weblate.org/projects/jamulus/installation-for-ios/)

* [Jamulus/Privacy-Statement](https://hosted.weblate.org/projects/jamulus/privacy-statement/)

* [Jamulus/Include-Shared-Commands](https://hosted.weblate.org/projects/jamulus/include-shared-commands/)

* [Jamulus/Include-Backing-Up](https://hosted.weblate.org/projects/jamulus/include-backing-up/)

* [Jamulus/Include-Client-Commands](https://hosted.weblate.org/projects/jamulus/include-client-commands/)

* [Jamulus/Navigation](https://hosted.weblate.org/projects/jamulus/navigation/)

* [Jamulus/FAQ](https://hosted.weblate.org/projects/jamulus/faq/)

* [Jamulus/Running-a-Server](https://hosted.weblate.org/projects/jamulus/running-a-server/)

* [Jamulus/Server-Troubleshooting](https://hosted.weblate.org/projects/jamulus/server-troubleshooting/)

* [Jamulus/Software-Manual](https://hosted.weblate.org/projects/jamulus/software-manual/)

* [Jamulus/Tips-Tricks-More](https://hosted.weblate.org/projects/jamulus/tips-tricks-more/)



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widgets/jamulus/-/1-index/horizontal-auto.svg)

ann0see commented on _translator-files/po/zh_CN/Running-a-Server.po:
We should purge duplicate E-Mails...

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/faq/
Translation: Jamulus/FAQ

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/faq/
Translation: Jamulus/FAQ

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/faq/
Translation: Jamulus/FAQ

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Translation: Jamulus/Server-Bandwidth
Translate-URL: https://hosted.weblate.org/projects/jamulus/server-bandwidth/

ignotus666 committed:
Revert "Fix conflict"

This reverts commit 8c9509d1242cd0c2d40e6f713560b16b5c3acadc.

ignotus666 committed:
Fix conflict

ignotus666 committed:
Merge branch 'next-release' into weblate-jamulus-1-index
